esmvaltool/diag_scripts/monitor/multi_datasets.py

```diff
@@ -998,17 +1071,23 @@ def _process_pyplot_kwargs(self, plot_type, dataset):
                     dataset,
                     f"pyplot_kwargs of {plot_type} '{func}: {arg}'",
                 )
-            getattr(plt, func)(arg)
+            if arg == 'None':
+                getattr(plt, func)()
+            else:
+                getattr(plt, func)(arg)
```

Comment:
This could be problematic in some cases: `matplotlib` actually uses the string `None`. What is you use case here? If this is really necessary, it's probably better to check for the "true" `None` with `if arg is None:`. In the recipe, `None` can be specified with `null`.

Reply:
Ah, okay, this was something I wanted to ask you. I had problems applying a pyplot_kwargs function with no argument. E.g. minorticks_on(). Do you have a suggest I was not aware of the `null` statement.

Reply:
I see, yes, in that case you need to call it without arguments, so it's probably best to use

```py
if arg is None:
```

in the code and

```yml
pyplot_kwargs:
  minorticks_on: null
```

in the recipe.